### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by cakarci**

### CODE REVIEW

**Comment by cakarci** [comment]
test

**Review by cakarci — COMMENTED** [review]
test

File: README.md [hunk 1]
```diff
@@ -49,69 +49,69 @@ A GitHub action that creates a workflow with **_four eyes principle_**
...
+            synchronize,
+            converted_to_draft,
+            ready_for_review,
+            locked,
+            unlocked,
+            review_requested,
+            review_request_removed,
+            auto_merge_enabled,
+            auto_merge_disabled
+          ]
```
Comment (pcakarci) [inline README.md:76]: oo

Comment (pcakarci) [reply README.md:76]: te

Comment (cakarci) [reply README.md:76]: tetds

Comment (cakarci) [reply README.md:76]: test

**Pull request merged**